readme.md

```diff
@@ -87,6 +87,17 @@ It honors both the `maxBuffer` and `encoding` options. The behavior changes slig
 - When `encoding` is set to anything else, it collects an array of strings. `maxBuffer` refers to the summed character lengths of every string in the array.
 
 
+## Errors
+
+When input stream emits an error, promise is rejected. The data gathered up until the error is passed via `buffer` property on the error object.
```

Comment:
Does this sound better?

If the input stream emits an `error` event, the promise will be rejected with the error. The buffer contents at the time of the error will be attached to the `buffer` property.